### PR TITLE
feat: rename Destructive settings to Settings

### PR DESCRIPTION
**Pull request by mariembencheikh**
#258

### CODE REVIEW

File: src/components/main/DeleteMemberSection.tsx [hunk 1]
```diff
@@ -23,24 +24,22 @@ const DeleteMemberSection = (): JSX.Element => {
...
             {t('PROFILE_DELETE_ACCOUNT_INFORMATION')}
           </Typography>
 
           <Button
-            variant="contained"
+            variant="outlined"
             color="error"
             onClick={() => setOpen(true)}
             disabled={isLoading}
+            sx={{ maxWidth: '200px' }}
```
Comment (spaenleh) [inline src/components/main/DeleteMemberSection.tsx:42]: Do not set maxWidth with pixel units. Instead use `maxWidth: 'min-content'` and in your case set `textWrap: 'no-wrap'` so the button adapts to the size of the content without wrapping on multiple lines.

**Review by spaenleh — CHANGES_REQUESTED** [review]
I am a bit puzzled by your update to the translation files. Can you explain what you want to do ?

File: src/langs/en.json [hunk 2]
```diff
@@ -31,6 +31,7 @@
   "PROFILE_DELETE_ACCOUNT_MODAL_INFORMATION": "Your account will be deleted permanently.",
   "PROFILE_DELETE_ACCOUNT_MODAL_CONFIRM_BUTTON": "Delete Permanently",
   "PROFILE_DELETE_ACCOUNT_MODAL_CANCEL_BUTTON": "Cancel",
+  "PROFILE_DESTRUCTIVE_SETTINGS_TITLE": "Destructive settings",
```
[the remark this answers is not in the document]
Comment (mariembencheikh) [reply src/langs/en.json:34]: This is the title for the delete account, there is other components

File: src/langs/ar.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
   "MAIN_MENU_AVATAR": "الصورة الرمزية",
   "MAIN_MENU_PASSWORD": "كلمة المرور",
   "MAIN_MENU_PUBLIC_PROFILE": "حساب عام",
-  "MAIN_MENU_DESTRUCTIVE_SETTINGS": "الإعدادات المدمرة",
+  "MAIN_MENU_SETTINGS": "الإعدادات المدمرة",
```
[the remark this answers is not in the document]
Comment (mariembencheikh) [reply src/langs/ar.json:15]: ah sorry, ok , maybe i'm not correctly understanding you

**Review by pyphilia — APPROVED** [review]

**Pull request merged**